yarn.lock

```diff
@@ -2527,7 +2527,7 @@ eslint-plugin-flowtype@^2.30.0:
     lodash "^4.15.0"
 
 "eslint-plugin-guardian-frontend@file:dev/eslint-plugin-guardian-frontend":
-  version "3.0.0"
+  version "4.0.0"
```

Comment:
Ah, I see the problem - it's a local file right? Does something like this work? `yarn add file:dev/eslint-plugin-guardian-frontend@^4.0.0 --dev` (stolen [here](https://github.com/yarnpkg/yarn/issues/3187)) ...